frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -75,19 +78,21 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({ state, hasOCS
   return (
     <>
       <ReviewItem title={t('ceph-storage-plugin~Backing storage')}>
-        {!isMCG && !isRhcs && (
+        {!isStandaloneExternal && (
```

Comment:
```suggestion
        {!isStandaloneExternal && !iRhcs &&(
```
Will fix with other reviews